rust/Cargo.toml.in

```diff
@@ -10,7 +10,10 @@ debug = true
 
 [features]
 lua = []
+external = ["ntp-parser"]
 
 [dependencies]
 nom = "~3.0"
 libc = "~0.2.0"
+
+ntp-parser = { version = "^0", optional = true }
```

Comment:
so cargo then figures out that when --enable-rust-external is used it should also get ntp-parser?